neqo-http3/src/connection.rs

```diff
@@ -1332,16 +1333,14 @@ mod tests {
         let events = neqo_trans_conn.events();
         let mut stop_sending_event_found = false;
         for e in events {
```

Comment:
This is a common pattern.  I was uncertain about whether to change this to use Iterator::any, which would be better for looking for the event.  But that would be lots of typing.